js/interface.js

```diff
@@ -2528,13 +2528,42 @@ var DynamicLists = (function() {
         _this.config.likesDataSourceId = '';
       }
       if (_this.config.social.bookmark && (!_this.config.bookmarkDataSourceId || _this.config.bookmarkDataSourceId === '')) {
-        // Create likes data source
+        // Create bookmarks data source
         bookmarksPromise = Fliplet.DataSources.create({
           name: appName + ' - Bookmarks',
-          bundle: true,
-          organizationId: organizationId // optional
+          bundle: false,
+          organizationId: organizationId, // optional
+          definition: {
+            views: [
+              {
+                name: 'userBookmarks',
+                bundle: true,
+                filter: {
+                  user: {
+                    $or: [
+                      // Guest users
+                      { uuid: 'session.uuid' },
+
+                      // Logged in users
+                      { email: 'session.email' },
+
+                      // Legacy support just in case
```

Comment:
this is made so that for LMA we can update the view definition manually and have existing bookmarks working.